sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,118 @@
+
+### Services
+
+| Name                    | Type                                                  | Description                                                                               |
+| ----------------------- | ----------------------------------------------------- | ----------------------------------------------------------------------------------------- |
+| `extrinsic_calibration` | `tier4_calibration_msgs::` `srv::ExtrinsicCalibrator` | Generic calibration service. The call is blocking until the calibration process finishes. |
+
+## Parameters
+
+### Core Parameters
+
+| Name                                          | Type                  | Default Value                    | Description                                                                                                                                               |
+| --------------------------------------------- | --------------------- | -------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `calib_rate`                                  | `double`              | `10.0`                           | The rate at which the calibration callback is invoked. This controls the frequency of calibration updates.                                                |
+| `base_frame`                                  | `std::string`         | `base_link`                      | The base_frame is used for visualization.                                                                                                                 |
+| `min_tag_size`                                | `double`              | `0.6`                            | The size of the AprilTag in meters (payload).                                                                                                             |
+| `max_tag_distance`                            | `double`              | `20.0`                           | Maximum allowed distance in meters from the camera to the tags.                                                                                           |
+| `max_allowed_homography_error`                | `double`              | `0.5`                            | AprilTag detection are discarded if the homography error is larger than `max_allowed_homography_error`.                                                   |
+| `use_receive_time`                            | `bool`                | `false`                          | Flag to determine whether to use the receive time instead of the header timestamps.                                                                       |
+| `use_rectified_image`                         | `bool`                | `true`                           | Flag to determine whether rectified images should be used in the calibration process.                                                                     |
```

Comment:
The explanation for `use_rectified_image` is wrong.
Just changing the value of this parameter does not change the image as the description implies. It determines if the image is treated as rectified or not

Reply:
fixed in https://github.com/tier4/CalibrationTools/commit/50eca415a164ec242cdb629eaacf73654c155162